File: CD/CP/Test_cases/C/loops.c
//  Find fibonacci
#include<stdio.h>

int main(){
    int i = 1;
    int fib =1;
    int target = 10;
    int temp = 0;
    while(i < target){
        temp = fib;
        fib += temp;
        i++;
    }
    write(fib);
    return 0;
}

File: sem1/CD/CP/Test_cases/Generated_Python/loops.py
def main () :
	i = 1
	fib = 1
	target = 10
	temp = 0
	while (i < target) :
		temp = fib
		fib += temp
		i+=1
	
	print(fib)
	return 0

if __name__ == '__main__':
	main()
